src/configuration/GPG/GnuPG/gpg.conf

```diff
@@ -205,8 +205,7 @@ keyserver hkp://keys.gnupg.net
 # photo-viewer "metamail -q -d -b -c %T -s 'KeyID 0x%k' -f GnuPG"
 
 
-personal-digest-preferences SHA256
-cert-digest-algo SHA256
-default-preference-list SHA512 SHA384 SHA256 SHA224 AES256 AES192 AES CAST5 ZLIB BZIP2 ZIP Uncompressed
-
+personal-digest-preferences SHA512
+cert-digest-algo SHA512
+default-preference-list AES256, CAMELLIA256, AES192, CAMELLIA192, AES, CAMELLIA128, TWOFISH, SHA512, SHA384, SHA256, BZIP2, ZLIB, ZIP 
```

Comment:
Oh yes, of course. I fixed the commas.

I tend to keep all of them in place. I think you are right that these algorithms are rarely used but I'm not aware that one of them is not secure anymore. Because [RFC4880](https://tools.ietf.org/html/rfc4880) defines TripleDES as a MUST implementation my point was to suggest as many better algorithms as possible to avoid a fallback to TripleDES.
Maybe AES256 and AES128 is not enough!?

`Uncompressed` is not really necessary, as it is also defined in RFC4880 as MUST implementation.